catalogs/catalog-jdbc-doris/src/main/java/org/apache/gravitino/catalog/doris/utils/DorisUtils.java

```diff
@@ -220,4 +217,26 @@ public static Distribution extractDistributionInfoFromSql(String createTableSql)
 
     throw new RuntimeException("Failed to extract distribution info in sql:" + createTableSql);
   }
+
+  private static int extractBucketNum(Matcher matcher) {
+    int bucketNum = 1;
+    if (matcher.find(5)) {
+      String bucketValue = matcher.group(5);
+      if (bucketValue.trim().toUpperCase().equals("AUTO")) {
+        // Use -1 to indicate auto bucket.
+        bucketNum = -1;
+      } else {
+        bucketNum = Integer.valueOf(matcher.group(5));
+      }
+    }
+    return bucketNum;
+  }
+
+  public static String revertDistributionBucket(int number) {
```

Comment:
The name seems a bit odd. Could you please rename it `toBucketNumberString` or something similar?

Reply:
@mchades @yuqi1129  PTAL, thanks.